redux/js/actions/index.js

```diff
@@ -0,0 +1,47 @@
+import shop from '../../../common/api/shop';
```

Comment:
No big deal, all examples currently have this.
Wouldn't want to tie this example too much to Browserify ecosystem.